docs/technical_reference/flowsheets/gac.rst

```diff
@@ -0,0 +1,103 @@
+Granular Activated Carbon
+=========================
+
+Introduction
+------------
+
+The simple granular activated carbon (GAC) flowsheet can be simulated to predict the performance of a GAC system to treat residual organics. This flowsheet can
+be useful to expedite the set-up, usage, and costing of a GAC system for conventional water treatment applications using the constant pattern homogeneous
+surface diffusion model (CPHSDM) model.
+
+Implementation
+--------------
+
+Only consisting of a single unit operation, the assumptions for the flowsheet are aligned with those detailed in the :doc:`GAC unit model documentation </technical_reference/unit_models/gac>`.
+The code-based naming of modeling objects for the inlets, outlets, units, and streams are shown in Figure 1.
+
+.. figure:: ../../_static/flowsheets/gac.png
+    :width: 500
+    :align: center
+
+    Figure 1. GAC flowsheet
+
+The following modeling components are used within the flowsheet:
+
+Property models:
+    * :doc:`/technical_reference/property_models/mc_aq_sol`
+Unit models:
+    * :doc:`/technical_reference/unit_models/gac`
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/feed`
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+Costing models:
+    * :doc:`/technical_reference/costing/watertap_costing`
+    * :doc:`/technical_reference/costing/gac`
```

Comment:
For consistency, I would use the same verbiage as other flowsheet documentation: "Documentation for each of the unit models can be found below"